app/vmalert/datasource/datasource.go

```diff
@@ -4,11 +4,17 @@ import (
 	"context"
 )
 
+// QuerierBuilder builds Querier with given params,
+// that will override corresponding default values.
```

Comment:
what are the `default values`? They're just implementation details, aren't they?

Reply:
removed mention about default values.